src/common/iop_order.c

```diff
@@ -391,7 +391,7 @@ static void _ioppr_insert_iop_before(GList **_iop_order_list, GList *history_lis
     if(found)
     {
       // set the iop_order
-      iop_order_new->iop_order = iop_order_prev + (iop_order_next - iop_order_prev) / 2.0;
+      iop_order_new->iop_order = iop_order_prev + (iop_order_next - iop_order_prev) / 8.0;
```

Comment:
not sure I get the reason here. `/ 2.0` was a bissection between previous and last module. What does the `/8.0` to the logic of the code ?